### PR TITLE
Lf probes

### PR DESCRIPTION
**Pull request by LucaFumis**
Short PR to address issue https://github.com/opentargets/issues/issues/2666 :

- remove reference column
- only show scores from Probes&Drugs
- remove chips and show only score
- remove redundant checks

### CODE REVIEW

**Comment by LucaFumis** [comment]
Flagging as draft until data is available and intended functionality is verified.

**Comment by LucaFumis** [comment]
The code is ready for review. The changes are pretty simple, however needs the latest backend/API changes in order to be tested in the preview.

**Comment by LucaFumis** [comment]
There have been some further changes - ready for @carcruz to review :)

**Comment by buniello** [comment]
Looks good Luca!
<details><summary>Browser metadata</summary>





```
Path:      /target/ENSG00000157764
Browser:   Chrome 105.0.0.0 on Mac OS 10.15.7
Viewport:  2076 x 1143 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-32--ot-platform.netlify.app%2Ftarget%2FENSG00000157764&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=105.0&resolution=2076x1143&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-32--ot-platform.netlify.app/target/ENSG00000157764) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63330521eba49b0009a20a1c&commentID=633408e1a8a8cb5fb382247d&resolution=resolved)

**Review by carcruz — APPROVED** [review]
Good to go

File: apps/platform/src/sections/target/ChemicalProbes/Body.jsx [hunk 1]
```diff
@@ -180,19 +105,16 @@ function Body({ definition, id, label: symbol }) {
       request={request}
       renderDescription={() => <Description symbol={symbol} />}
       renderBody={data => {
-        // sort probes manually as we need a double custom sort based on quality and origin
-        const sortedProbes = _.sortBy(data.target.chemicalProbes, [
```
Comment (ireneisdoomed) [inline apps/platform/src/sections/target/ChemicalProbes/Body.jsx:184]: Hi @LucaFumis, sorry I didn't read your update before it was merged.
Ideally, we still want to sort on these parameters - especially in the case of probes of high quality. I meant to add the score sorting on top of the other two. 
I don't think it is a reason to halt the release process, however.

**Pull request merged**